inference-engine/src/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -160,6 +159,7 @@ static bool replace_squeeze_unsqueeze(const std::shared_ptr<Node>& node) {
 
     // skip if reshape is nop
     if (reshape->get_input_partial_shape(0).same_scheme(shape_ps)) {
+        copy_runtime_info({input, node->output(0).get_node_shared_ptr()}, node->output(0).get_node_shared_ptr());
```

Comment:
Shouldn't we copy runtime info on else branch as well?

Reply:
The replace_output_update_name and replace_node_update_name methods copy runtime info. An additional copy_runtime_info is required in an exceptional situation.